internal/source/kubernetes/router.go

```diff
@@ -224,7 +222,7 @@ func (r *Router) setEventRouteForRecommendationsIfShould(routeMap *map[config.Ev
 	// Override route and get all these events for all namespaces.
 	// The events without recommendations will be filtered out when sending the event.
 	for i, r := range (*routeMap)[eventType] {
-		recommRoute.namespaces = r.namespaces
+		recommRoute.namespaces = resourceNamespaces(cfg.Namespaces, *r.namespaces)
```

Comment:
can we check if `r is nil first? just to be on the safe side. Thanks!

Reply:
I changed func signature so that both args are pointers and we check for in in one place.